students/kulon_aleksander/lesson_11/filling_in_the_gaps.py

```diff
@@ -0,0 +1,47 @@
+import re
+import os
+import shutil
+import argparse
+import sys
+
+pattern = 'spam00'
+path = '../lesson_11/testfolder/'
+
+
+def main():
+    sorted_numbers = []
+
+    parser = argparse.ArgumentParser(description='Filling gaps.')
+
+    parser.add_argument(
+        '--pattern', action="store",
+        dest='pattern',
+        default=pattern,
```

Comment:
```default``` flag shouldn't be used with ```required``` flag set to true. It hasn't sense😄 Have a look at this short descriptions: https://docs.python.org/2/library/argparse.html#argparse.ArgumentParser.add_argument